CoreNamedSelectionView/src/au/gov/asd/tac/constellation/views/namedselection/NamedSelectionTopComponent.java

```diff
@@ -15,6 +15,7 @@
  */
 package au.gov.asd.tac.constellation.views.namedselection;
 
+import au.gov.asd.tac.constellation.views.*;
```

Comment:
Was everything from views required? Better to only import the stuff that is specifically needed

Reply:
I've just fixed this, I'm not sure why it keeps automatically adding all when I add the import. I'll keep an eye on this for the future.